### ISSUE TITLE
Payroll Tax + Income Tax buttons on difference table don't work correctly

### ISSUE THREAD

**Issue by martinholmer** [OpenSourcePolicyCenter/PolicyBrain#644]
The buttons currently say `Payroll` and `Income`, but it would be far better to label the buttons as `Payroll Tax` and `Income Tax`.  I'm going to assume that change will be made soon, and in the rest of this issue refer to the buttons using those suggested labels.

You can see the problems by using these buttons at [this static results page](http://www.ospc.org/taxbrain/16252/).

Here are a few click sequences that I tried:
```
Starting with the default display presented when clicking on Difference Table button:

paytax=ON  + inctax=ON   ==> showing combined taxes which is OK

Then click on the Payroll Tax button which produces this:

paytax=OFF + inctax=ON   ==> showing income taxes which is OK
```
The above sequence is just fine, but the follow sequence is not correct:
```
Starting with the default display presented when clicking on Difference Table button:

paytax=ON  + inctax=ON   ==> showing combined taxes which is OK

Then click on the Income Tax button which produces this:

paytax=ON  + inctax=OFF  ==> still showing combined taxes which is INCORRECT

Then click on the Income Tax button which produces this:

paytax=ON  + inctax=ON   ==> showing payroll taxes which is INCORRECT
```
The GUI also allows a user to turn OFF both the `Payroll Taxes` and `Income Taxes` buttons.
Surely that combination doesn't make any sense.

**Comment by MattHJensen** [comment]
@martinholmer, thanks for the bug report. 

cc @GoFroggyRun

**Comment by GoFroggyRun** [comment]
Working on it. 

cc @martinholmer @MattHJensen

**Comment by martinholmer** [comment]
Issue #644 has been resolved in TaxBrain 1.0.3.  @GoFroggyRun, thanks for your work on pull request #648, which fixed this bug.
